pkg/platform/model/vcs.go

```diff
@@ -717,11 +717,11 @@ func languageCommitMessage(op Operation, name, version string) string {
 	var msgL10nKey string
 	switch op {
 	case OperationAdded:
-		msgL10nKey = locale.T("commit_message_add_language")
+		msgL10nKey = "commit_message_add_language"
```

Comment:
nit:
```suggestion
		msgL10nKey = "commit_message_added_language"
```
To keep these all in the same tense. The same goes for `packages` and `platforms`